src/R/comparative-charts.R

```diff
@@ -0,0 +1,45 @@
+library(dplyr)
+library(ggplot2)
+
+
+# DATA
+
+## cost
+baseline_cost_data = "data/total_cost/TH-VersionA-baseline-29-03-18-Cost.csv"
```

Comment:
Could we make these args to a function which just point at two directories that contain files ending in `Cost.csv` and `Count.csv`?

Reply:
done